ONSdigital/rm-sample-service
# adding party id code and modifiying database

# What and why?
The party id needs to be attached to a sample as part of the new design.

# How to test?
Run the acceptance tests and check the database. The sampleunit table should have party IDs filled in.

# Trello
[Card](https://trello.com/c/vnhoLDvM)
